src/docs/implementations/datadog.adoc

```diff
@@ -63,6 +63,14 @@ management.metrics.export.datadog:
     step: 1m
 ----
 
+`DatadogConfig.hostTag()` specifies a tag key that will be mapped to the `host` field when shipping metrics to Datadog.
```

Comment:
We should link to the Datadog documentation that mentions the `host` field: https://docs.datadoghq.com/api/v1/metrics/#submit-metrics
I'm not sure if there's somewhere better in the documentation describing the `host` field.

Reply:
@shakuzen Thanks for the feedback! I updated as suggested.